holder.go

```diff
@@ -351,6 +359,42 @@ func (h *Holder) flushCaches() {
 	}
 }
 
+// setFileLimit attempts to set the open file limit to the FileLimit constant defined above.
+func (h *Holder) setFileLimit() {
+	lim := &syscall.Rlimit{}
+	var setLimit uint64
+	if err := syscall.Getrlimit(syscall.RLIMIT_NOFILE, lim); err != nil {
+		h.logger().Printf("ERROR checking open file limit: %s", err)
+		return
+	}
+	if lim.Max < FileLimit {
+		h.logger().Printf("WARNING: open file limit max is %d. Please consider running \"ulimit -n %d\" before starting Pilosa to avoid \"too many open files\" error.", lim.Max, FileLimit)
```

Comment:
Maybe I can add a section to the documentation about these caveats and provide the URL to that. Also, I have no idea how this works in other OSs (Windows and BSD), so just describing the problem in more detail would be good to help people along.